hashicorp/boundary-ui
# feat: 🎸 Add section for client agent in settings

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14780

# Description
Adds a section for the client agent.

## Screenshots (if appropriate)
<img width="936" alt="Screenshot 2024-08-15 at 7 31 55 PM" src="https://github.com/user-attachments/assets/a64e8dc5-15b0-48dd-8b8f-d06fcf241b98">
<img width="947" alt="Screenshot 2024-08-15 at 7 16 57 PM" src="https://github.com/user-attachments/assets/3de14563-f428-4ce2-a615-33080bd3bdc4">

## How to Test
First test the settings page with no client agent, there should not be a section for it if a user isn't running it. Turn on the daemon and then everything should say running. Try pausing and unpausing. Use an CE controller and confirm that it returns an error that says it's not compatible. 

## Checklist
